topic_resources/capstone/capstone.md

```diff
@@ -1,51 +1,27 @@
 # Capstone Project
+The Capstone Project is the largest undertaking you'll experience in the Ada classroom. Not only is it a month long, but you will be your own product owner. It will be up to you to decide (with input from instructors, mentors, and TAs) what gets built when (and how).
 
-The Capstone Project is the largest undertaking you'll experience in the Ada classroom.
+### Teams
+For the capstone project, you have the option to work individually or in a pair. If you do decide to work individually, you will have another individual with whom you will work with throughout the capstone as a sounding board and support system even though you will be working on different projects.
 
-Not only is it a month long, but you will be your own product owner. It will be up to you to decide (with input from instructors, mentors, and TAs) what gets built when (and how).
+### Goals
+- Demonstrate self-direction, time management, and independent learning
+- Learn and implement new technologies
+- Complete a product lifecycle from conception to delivery
+- Utilize agile practices learned to assist in project completion
```

Comment:
This was something that was brought up in feedback: not being clear about what the goals of the capstone were/are. So just a note to us to remember to go over these goals with the students at the beginning (and then revisiting a couple times during capstone).

Also making sure that these are the goals that we really want.